### PR TITLE
Change reset page errors to match sign-in page

### PR DESCRIPTION
**Pull request by cobbspur**
No Issue
- Takes the inline validation messages and displays below form
- Removes unnecessary gh-error-message components from signin and reset pages
  - Returns error messages to sign-in validations

### CODE REVIEW

File: core/client/app/templates/reset.hbs [hunk 1]
```diff
@@ -3,22 +3,16 @@
...
-                    {{input value=newPassword class="gh-input password" type="password" placeholder="Password" name="newpassword" autofocus="autofocus" focusOut=(action "validate" "newPassword")}}
-                    <div class="pw-strength">
-                        <div class="pw-strength-dot"></div>
-                        <div class="pw-strength-dot"></div>
-                        <div class="pw-strength-dot"></div>
-                        <div class="pw-strength-dot"></div>
-                        <div class="pw-strength-dot <!--pw-strength-activedot-->"></div>
-                    </div>
-                    {{gh-error-message errors=errors property="newPassword"}}
+                    {{input value=newPassword class="gh-input password" type="password" placeholder="Password" name="newpassword" autofocus="autofocus" }}
```
Comment (ErisDS) [inline core/client/app/templates/reset.hbs]: These should be `{{gh-input}}` and not have class `gh-input` I believe

**Comment by cobbspur** [comment]
updated PR

**Comment by ErisDS** [comment]
LGTM :+1: cc @kevinansfield good to go?

**Comment by kevinansfield** [comment]
:+1:

**Pull request merged**